tests/test_stommel.py

```diff
@@ -0,0 +1,141 @@
+from parcels import Grid, Particle, JITParticle
+from parcels import AdvectionRK4, AdvectionEE, AdvectionRK45
+from argparse import ArgumentParser
+import numpy as np
+import math
+import pytest
+from datetime import timedelta as delta
+
+
+method = {'RK4': AdvectionRK4, 'EE': AdvectionEE, 'RK45': AdvectionRK45}
+
+
+def ground_truth(lon, lat):
+    day = 11.6
+    r = 1 / (day * 86400)
+    beta = 2e-11
+    a = 2000000
+    e_s = r / (beta * a)
+    psi = (1 - np.exp(-lon * math.pi / 180 / e_s) - lon *
+           math.pi / 180) * math.pi * np.sin(math.pi ** 2 * lat / 180)
+    return psi
+
+
+def stommel_grid(xdim=200, ydim=200):
+    """Generate a grid encapsulating the flow field consisting of two
+    moving eddies, one moving westward and the other moving northwestward.
+
+    The original test description can be found in: K. Doos,
+    J. Kjellsson and B. F. Jonsson. 2013 TRACMASS - A Lagrangian
+    Trajectory Model, in Preventive Methods for Coastal Protection,
+    T. Soomere and E. Quak (Eds.),
+    http://www.springer.com/gb/book/9783319004396
+    """
```

Comment:
This reference here is incorrect. @Abobie could you suggest the correct reference?